src/decoder.c

```diff
@@ -186,6 +187,14 @@ void tfi_pip_decoder(struct decoder *d) {
   stats_finish_timing(STATS_DECODE_TIME);
 }
 
+static struct console_feed_node decoder_feed_nodes[] = {
```

Comment:
should i move this to console.c?

Reply:
I decided to keep everything in console, though we can leave open the option to decentralize it later.